tests/Sinch.Tests/e2e/Conversation/TemplatesTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Sinch.Conversation.Messages.Message;
+using Sinch.Conversation.TemplatesV2;
+using Xunit;
+
+namespace Sinch.Tests.e2e.Conversation
+{
+    public class TemplatesTests : TestBase
+    {
+        private static readonly DateTime DefaultTime = Helpers.ParseUtc("1970-01-01T00:00:00Z");
+
+        private readonly Template _template = new Template()
+        {
+            Id = "template_id",
+            Description = "template",
+            CreateTime = DefaultTime,
+            DefaultTranslation = "en-US",
+            UpdateTime = DefaultTime,
+            Version = 3,
+            Translations = new List<TemplateTranslation>()
+            {
+                new TemplateTranslation(new TextMessage("hello ${a}"))
+                {
+                    Version = "1", // is this really a string?
```

Comment:
Is this comment still necessary?

Reply:
yeah, I'm still confused and this confusion is for future me

Reply:
The documentation says clearly it's a string. Maybe you can update the comment to clear the reader's doubts with "The property is really a string, contrary to the template version which is a integer"